api/app/views/spree/api/variants/small.v1.rabl

```diff
@@ -14,4 +14,4 @@ child :option_values => :option_values do
   attributes *option_value_attributes
 end
 
-child(:images => :images) { extends "spree/api/images/show" }
+child(:display_images => :images) { extends "spree/api/images/show" }
```

Comment:
@jhawthorn good point, it would not work for the variant search route without providing a similar n+1 query

re the images, it is only one-per-rule-per-product. so for us where we only have rules per color it is more like O(log(n)), but for people who do one rule per variant, it would be more like O(n). We do already have a another n+1 query in the products endpoint (variants -> stock items), so I think some performance consideration is overdue on that endpoint either way. Aside from denormalization or putting these images at the product level (and then having search use another field that does the n+1), I'm currently at a loss for potential solutions to the slow query. 

While it is (very) helpful to poke holes, I am interested to hear some potential solutions and/or preferences of these trade-offs from you as well.